README.md

```diff
@@ -1 +1,976 @@
-# balena-cli-advanced-masterclass
\ No newline at end of file
+balenaCLI Advanced Masterclass
+==============================
+
+# Prerequisite Classes
+
+This masterclass builds upon knowledge that has been taught in previous classes.
+To gain the most from this masterclass, we reommend that you first undertake
+the following masterclasses:
+
+* [balenaCLI Masterclass](https://github.com/balena-io-projects/balena-cli-masterclass)
+
+**Masterclass Type:** Core
+**Maximum Expected Time To Complete:** 120 minutes
+
+# Introduction
+
+The balena Command Line Interface (balenaCLI) utility consists of a number
```

Comment:
This change is just for consistency with how we name/call the product (https://github.com/balena-io/balena-cli)

Reply:
Hmm, I'd rather change this to 'balena CLI' as how we refer to it in the docs, but not the package name. I know the package name is very similar, but 'balena CLI' refers to the CLI method of operating with balena, rather than the package name (and if we ever changed the package name, it's still valid).

Reply:
As long as it's consistent, that's fine with me!